clients/admin-ui/src/pages/dataset/new/index.tsx

```diff
@@ -28,7 +28,6 @@ const NewDataset: NextPage = () => {
         <Box>
           <Button
             onClick={() => setGenerateMethod("yaml")}
-            ghost={generateMethod === "yaml"}
```

Comment:
Prior to migrating, the button was in an "active"/pressed state while the YAML form was open on the page.  Ant doesn't allow you to set the pressed state in props, but I'm not sure it's really necessary here anyway.